cli

```diff
@@ -145,3 +181,9 @@ if __name__ == "__main__":
             guid = user.get("id", "No GUID")
             email = user.get("userName", "Not Found")
             print(f"Email: {email}, GUID: {guid}")
+
+    if args.command == "list-last-logon":
+        summary = bot.get_all_user_last_logon(
+            days_ago=0, days_range=365, summary_title=args.title
```

Comment:
looks like we're ignoring the cli args here
```suggestion
            days_ago=args.days_ago, days_range=args.day_range, summary_title=args.title
```

Reply:
Added this with a recent commit